File: Solutions/2974/2974.js
function numberGame(nums) {
  nums.sort((a,b)=>a-b); // Sort in ascending order
  for (let i = 0; i < nums.length; i += 2) {
    [nums[i], nums[i + 1]] = [nums[i + 1], nums[i]]; // Swap elements
  }
  return nums;
}

File: Solutions/2974/2974.go
func numberGame(nums []int) []int {
  sort.Ints(nums) // Sort in ascending order
  for i := 0; i < len(nums)-1; i += 2 {
    nums[i], nums[i+1] = nums[i+1], nums[i] // Swap elements
  }
  return nums
}
